test/src/test/java/hudson/model/ProjectTest.java

```diff
@@ -194,15 +190,17 @@ public void testGetSomeWorkspace() throws Exception{
         getFilePath = true;
         assertNotNull("Project should have any workspace because WorkspaceBrowser find some.", p.getSomeWorkspace());
         getFilePath = false;
-        p.getBuildersList().add(new Shell("echo ahoj > some.log"));
+        String cmd = "echo ahoj > some.log";
+        p.getBuildersList().add(Functions.isWindows() ? new BatchFile(cmd) : new Shell(cmd));
```

Comment:
The command itself works fine on both batch and shell, so I agree, XShell might help, but it seems a little overkill to pull in that plugin for a few tests.